src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+function Header() {
+  return (
+    <header>
+      <h1>todos</h1>
+    </header>
+  )
+}
+
+export default Header
```

Comment:
Hello, it would be a great idea to end this line with a semi-colon. Kindly have a look at list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for a refresher.